rotkehlchen/data_migrations/migrations/migration_12.py

```diff
@@ -0,0 +1,50 @@
+import logging
+from typing import TYPE_CHECKING
+
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.data_import.importers.constants import ROTKI_EVENT_PREFIX
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+
+if TYPE_CHECKING:
+    from rotkehlchen.data_migrations.progress import MigrationProgressHandler
+    from rotkehlchen.rotkehlchen import Rotkehlchen
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+PREFIX = f'{ROTKI_EVENT_PREFIX}_%'
```

Comment:
Shouldn't `ROTKI_EVENT_PREFIX` be hardcoded here instead of using the variable?

Reply:
What do you mean? What if it changes?

Reply:
yes

Reply:
Ah I see. Okay yeah. makes sense, but will add a comment for it since I was trying to find places we hard code those and replace with variables (except for db upgrades and migrations)